lib/libzrepl/data_conn.c

```diff
@@ -441,8 +473,7 @@ uzfs_zvol_timer_thread(void)
 	time_t min_interval;
 	time_t now, next_check;
 
-	mutex_init(&timer_mtx, NULL, MUTEX_DEFAULT, NULL);
-	cv_init(&timer_cv, NULL, CV_DEFAULT, NULL);
+	init_zrepl();
```

Comment:
Would be great if we rename it to init_zrepl_timer().

Reply:
we can be adding more to this. Actually, more will come when we move code from zrepl binary to zrepl library.